### ISSUE TITLE
Final Peer Review-cl2636

### ISSUE THREAD

**Issue by CheweiLou** [caperstar/ORIE5741#12]
The project aims to predict applications' credit score and help to make decisions on credits. The data comes from the Home Mortgage Disclosure Act provided by the FFIEC. The model, ideally, could enable quick determination of whether the application should be approved.

 Things I like: 
1. The box plot and the feature engineering dealing with different kinds of outliers are reasonable. Besides, the data balancing is a good idea.
2. Apart from accuracy, they use another indicator recall score to evaluate models, which is not included in the class. And the recall seems meaningful in this application.
3. The presentation is clear and organized.
4. The feature importance is helpful in showing what aspects are crucial in credits decisions.

Improvements:
1. It seems that they use all the dataset to train the model without splitting the dataset into train/test sets. Thus, even if the accuracy/recall score is 95% with Random Forest, it is still on the training set. We still do not know the model performance on the new data and it could be overfitting. So it is better to test the model on a test set to evaluate the performance.
2. It is better to include more exploratory data analysis in the report (e.g. descriptive statistic, histograms, etc.) to present a clearer and more specific view of the dataset.
3. In terms of fairness issue, the model excludes such protected attributes as gender, race, etc., which means they accept the notion of unawareness, but this will also reduce the accuracy. Thus, other notions like "equalized odds" could be used, in which case the protected attributes could also be incorporated into the model.

**Comment by caperstar** [comment]
For our project, we actually split the dataset before train the model
